stories/exercises/05-customize-theme-tokens.mdx

```diff
@@ -0,0 +1,120 @@
+import { Meta, ColorPalette, ColorItem } from "@storybook/blocks";
+
+# 05. Customize with theme tokens
+
+Now that we have content on our landing page, we're going look at how we can make wide sweeping changes using USWDS Design Tokens and theme settings.
+
+You can view the available theme settings by visiting the [USWDS Settings Table](https://designsystem.digital.gov/documentation/settings/#general-settings-2).
+
+Lets start by updating some of our [theme color tokens](https://designsystem.digital.gov/documentation/settings/#color-settings-2). USWDS theme color tokens are designed to be useful and effective for small and large projects.
+
+1. Pick a color palette and copy SCSS variables.
+
+  <h2>Colors</h2>
+  <h3>Earth tones</h3>
+  <ColorPalette>
+    <ColorItem
+      title="Base"
+      subtitle="Base colors"
+      colors={{
+        'yellow-5v': '#fff5c2',
+        'yellow-10': '#f5e6af',
+        'yellow-30v': '#ddaa01',
+        'yellow-80v': '#422d19',
+      }}
+    />
+    <ColorItem
+      title="Primary"
+      colors={{
+        'green-warm-60v': '#5a6613',
+        'green-warm-70v': '#4b4e10',
+      }}
+    />
+    <ColorItem
+      title="Accent"
+      colors={{
+        'gold-30v': '#e5a000',
+      }}
+    />
+  </ColorPalette>
+
+  Theme setting values
+
+  ```scss
+  $theme-color-primary: "green-warm-60v",
+  $theme-color-primary-dark: "green-warm-70v",
+  $theme-color-primary-darker: "green-warm-80v",
+  $theme-color-base-lightest: "yellow-5v",
+  $theme-color-base-lighter: "yellow-10",
+  $theme-color-base-light: "yellow-30v",
+  $theme-color-base-darkest: "yellow-80v",
+  $theme-color-accent-cool: "gold-30v",
+  ```
+
+  <h3>Oceanic</h3>
+  <ColorPalette>
+    <ColorItem
+      title="Base"
+      subtitle="Base colors"
+      colors={{
+        'blue-cool-5': '#e7f2f5',
+        'blue-cool-10': '#dae9ee',
+        'blue-cool-30': '#82b4c9',
+        'blue-cool-80': '#14333d',
+      }}
+    />
+    <ColorItem
+      title="Primary"
+      colors={{
+        'cyan-60': '#2a646d',
+        'cyan-70v': '#0e4f5c',
+      }}
+    />
+    <ColorItem
+      title="Accent"
+      colors={{
+        'mint-cool-30': '#6fbab3',
+      }}
+    />
+  </ColorPalette>
+
+  Theme setting values
+`
+  ```scss
+  $theme-color-primary: "cyan-60",
+  $theme-color-primary-dark: "cyan-70v",
+  $theme-color-primary-darker: "cyan-80v",
+  $theme-color-base-lightest: "blue-cool-5",
+  $theme-color-base-lighter: "blue-cool-10",
+  $theme-color-base-light: "blue-cool-30",
+  $theme-color-base-darkest: "blue-cool-80",
+  $theme-color-accent-cool: "mint-cool-30",
+  ```
+
+2. Visit `uswds-theme.scss`.
```

Comment:
**suggestion:** Using `1` will allow you to not worry about the specific step. Let's you easily move steps around or create new ones.

```suggestion
1. Visit `uswds-theme.scss`.
```

Reply:
Resolved in [b78561b](https://github.com/Bixal/storybook-rapid-prototyping/pull/13/commits/b78561b80cae38c00d9de5a1dd2e882c5416f184)